docs/design-systems/index.md

```diff
@@ -1,9 +1,100 @@
+
+It provides a streamlined approach to email design. Users can create reusable blocks and templates, define variables for global customization, and associate design systems with specific projects. This makes the process of managing email styles and content consistent and efficient across all campaigns.
+
+## Design System Principles
+The design system follows Atomic Design, breaking emails into reusable components for consistency and scalability.
+
+- Atoms – Basic elements like colors and text styles.
+- Molecules – Functional groups, such as buttons with icons.
+- Organisms – Larger sections like headers or footers.
+- Templates – Define email layouts.
+- Pages – Final email designs for campaigns.
+
+This modular approach keeps email designs structured, flexible, and easy to manage.
+
+## Components of the Design System
+Every design system in bluefox.email includes:
+
+- [Variables](./variables)
+- [Blocks](./blocks)
+- [Templates](./templates)
+
+Everything in a design system is built to be reusable and scalable, ensuring that all emails within a project maintain a consistent style and structure
+
+## Navigating to Design System
+
+You can find the design system section by clicking on it's icon on the left-hand side.
+
+![A screenshot of the design system section.](./design-system-icon.webp)
+
+## Creating a New Design System
+To create a new design system, navigate to the design system section and click on "Create" You can choose to start from scratch or use a default design system as your base. Once created, you can add variables, blocks, and templates to build your system.
+
+![A screenshot of the design system create dialog.](./design-system-create-dialog.webp)
+
+## Associating a Design System with a Project
+Every project must be linked to a design system to ensure consistency in design and functionality. When you create a new project, you will have the option to associate it with an existing design system. This association allows the project to inherit templates, blocks, and variables from the design system, making the design process more efficient.
+
+::: info Note
+A project can only be associated with one design system at a time. If you need to switch design systems, you can do so in the project settings, but this may affect existing templates and variables.
+:::
+
+
+### Overriding Variables at the Project Level
+While the design system provides a consistent foundation, you might need to customize certain elements for specific projects. You can override variable values at the project level without affecting the main design system. For instance, if the default color variable "primary" is set to red in the design system, you can override it to blue in a specific project. This change will only apply to the project in question, leaving the original design system unchanged.
```

Comment:
I would add the logo as well as an example, that makes it even easier to understand together with the primary color